zerver/views/typing.py

```diff
@@ -62,3 +70,40 @@ def send_notification_backend(
         check_send_typing_notification(user_profile, user_ids, operator)
 
     return json_success(request)
+
+
+@typed_endpoint
+def send_message_edit_notification_backend(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    *,
+    operator: Annotated[Literal["start", "stop"], ApiParamConfig("op")],
+    message_id: Json[int],
+) -> HttpResponse:
+    message = access_message(user_profile, message_id)
+    validate_user_can_edit_message(user_profile, message, 0)
```

Comment:
Let's pass the 0 as `edit_limit_buffer=0` -- it's a lot more readable spelled out that way.